client/my-sites/customer-home/cards/launchpad/keep-building.tsx

```diff
@@ -3,11 +3,19 @@ import { useLaunchpad } from '@automattic/data-stores';
 import { Launchpad, Task } from '@automattic/launchpad';
 import { useTranslate } from 'i18n-calypso';
 import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 import './style.scss';
 
+function recordTaskClickTracksEvent( is_completed: boolean, task_id: string ) {
+	recordTracksEvent( 'calypso_launchpad_task_clicked', {
```

Comment:
Yay Tracks events!

This should be `click` instead of `clicked`. The FG has more info about naming conventions for Tracks events.

We also need to include the task list name (`keep-building`) and the surrounding context (probably `customer-home`) in the event properties. The Tracks event task for reference: https://github.com/Automattic/wp-calypso/issues/77616

Reply:
This is actually the same Tracks event from the launchpad on stepper :) 

Should we create a new one?

Reply:
Just bringing the Slack discussion here for posterity, we'll keep the same event name since it already exists and we don't want to introduce more confusion, but we should definitely add the context and task list id to the event properties.

Reply:
Related PR https://github.com/Automattic/wp-calypso/pull/78181